posthog/models/user_scene_dashboard_choice.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+from .utils import UUIDModel
+
+
+class UserSceneDashboardChoice(UUIDModel):
```

Comment:
Potentially bike shedding...

Once we have Notebooks I could imagine the same functionality. Do we want to pre-emptively name the model something more generic? `UserSceneRelationChoice` or something?

Reply:
Thinking about SharingConfiguration being a similar thing